eng/common/scripts/git-diff-changes.ps1

```diff
@@ -11,11 +11,15 @@
 
   .PARAMETER TargetCommittish
   The branch committish PR targets to merge into.
+
+  .PARAMETER IncludeRegex
+  The regex of included files.
 #>
 [CmdletBinding()]
 param (
   [string] $SourceCommittish = "${env:BUILD_SOURCEVERSION}",
-  [string] $TargetCommittish = ("origin/${env:SYSTEM_PULLREQUEST_TARGETBRANCH}" -replace "refs/heads/")
+  [string] $TargetCommittish = ("origin/${env:SYSTEM_PULLREQUEST_TARGETBRANCH}" -replace "refs/heads/"),
+  [string] $IncludeRegex = ""
```

Comment:
```suggestion
  [string] $diffPath = ""
```

Reply:
This isn't really a regex so I want to avoid any confusion it is just a path

Reply:
It is used in many ways:
1. Scan against all markdown: git diff -- '*.md'
2. Scan against on folder: git diff -- 'eng/common/*'

It is much like regex syntax.

Reply:
Yes it is similar to a regex but it is different. I think we should try to name these parameters for what they are as we pass them to git and they are just paths in the git diff documentation.